sql/DB_Schema_v08.sql

```diff
@@ -86,6 +86,7 @@ CREATE TABLE `running_modules` (
   `position` int(8) DEFAULT NULL,
   `teacher1_id` int(8) DEFAULT NULL,
   `teacher2_id` int(8) DEFAULT NULL,
+  `description` VARCHAR(255) DEFAULT NULL,
```

Comment:
adding a required raw to the database to solve one of the issues about `adding a new class from the timeline`